src/Classes/ServiceAPI/MyRadio_Scheduler.php

```diff
@@ -131,7 +131,14 @@ public static function getTerms()
             ORDER BY start ASC'
         );
     }
-
+    
```

Comment:
muh trailing

Reply:
Huh, my code editor is setup to do that...